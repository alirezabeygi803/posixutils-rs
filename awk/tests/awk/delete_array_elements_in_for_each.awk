BEGIN {
	array["first"] = 1
	array["second"] = 2
	array["third"] = 3
	for (key in array) {
		print key, "->", array[key]
		delete array[key]
	}
	print "Array after deletion:"
	for (key in array) {
		print key, "->", array[key]
	}
}