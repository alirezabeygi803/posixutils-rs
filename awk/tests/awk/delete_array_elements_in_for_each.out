first -> 1
second -> 2
third -> 3
Array after deletion:
